src/main/java/com/nitnelave/CreeperHeal/block/CreeperChest.java

```diff
@@ -37,6 +39,7 @@
     {
         super(blockState);
         chest = getBlock();
+        data = chest.getData();
```

Comment:
If I'm not mistaken, this API is deprecated, there should be another one for explicitly getting the direction of the chest.